### PR TITLE
#138: Read and write StatusMessage in Saml2Response.

### PR DESCRIPTION
**Pull request by jonsagara**
I believe this satisfies the requirements for reading and writing `StatusMessage` in `Saml2Response`. Will you please review?

Original issue: #138 

Thanks!

### CODE REVIEW

**Review by fmaeseele — COMMENTED** [review]
See comment in Saml2Response.cs file review

File: src/ITfoxtec.Identity.Saml2/Request/Saml2Response.cs
[the remark this answers is not in the document]
Comment (jonsagara) [reply src/ITfoxtec.Identity.Saml2/Request/Saml2Response.cs:1]: When I try to add that, I get the following error:

```
Error	CS8370	Feature 'nullable reference types' is not available in C# 7.3. Please use language version 8.0 or greater.	ITfoxtec.Identity.Saml2 (net462), ITfoxtec.Identity.Saml2 (net48)	
```

Do I need to use conditional compilation?

[the remark this answers is not in the document]
Comment (jonsagara) [reply src/ITfoxtec.Identity.Saml2/Request/Saml2Response.cs:1]: I tried conditional compilation, and then got the following warning:

```
Warning	CS8632	The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
```

I think I would need to conditionally enable nullable reference types for the project (at a minimum), which is a bigger task than I feel comfortable doing at this point. Are you okay with leaving it as-is? (With no nullable annotation)

**Comment by Revsgaard** [comment]
Thanks, I'll look into the code :)

**Comment by Revsgaard** [comment]
Thank you for the code, it looks good. It is merged into master and will be included in the next release.

**Comment by Revsgaard** [comment]
Released in [4.8.7-beta2](https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/releases/tag/4.8.7-beta2).

**Comment by jonsagara** [comment]
Thank you!

**Pull request merged**